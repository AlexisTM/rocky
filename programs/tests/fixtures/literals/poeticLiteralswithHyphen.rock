
love is all-consuming
shout love

my story is a never-ending fairy-tale-like romance-involving child fable
say my story

the dictator was power-hungry
whisper the dictator
