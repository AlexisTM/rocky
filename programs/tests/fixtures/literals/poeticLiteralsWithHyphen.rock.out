3
125755
2